site/cypress/e2e/Cypress-UI/pronouns_change.spec.js

```diff
@@ -12,18 +9,15 @@ describe('Tests cases abut changing user pronouns', () => {
         cy.get('#pronouns_val').click();
         cy.get('#user-pronouns-change').as('e');
 
-        //save old pronouns
-        cy.get('@e').then(($pronounsInput) => {
-            oldPronouns = $pronounsInput.val();
-        });
-
-        //type in new pronouns
+        //type in new pronouns and check display in forum option
         cy.get('button[aria-label="Clear pronoun input"]').click(); //clear input using trash can
         cy.get('@e').type('They/Them');
+        cy.get('#pronouns-forum-display').check();
         cy.get('#edit-pronouns-submit').click();
 
-        //ensure pronouns changed on page
+        //ensure pronouns and display option changed on page
         cy.get('#pronouns_val').contains('They/Them');
```

Comment:
Same comment about the should('contain', 'text') for this line and the one after.